Frontend/src/router.js

```diff
@@ -25,6 +26,11 @@ let router = new VueRouter({
       // which is lazy-loaded when the route is visited.
       component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
     },
+    {
+      path: '/app-register',
+      name: 'app register',
+      component: AppRegister
```

Comment:
[non-blocking] Can we name this something other than app-register? I feel like this route is confusing.